src/dev/flang/ast/AbstractFeature.java

```diff
@@ -464,7 +464,7 @@ boolean hasOpenGenericsArgList(Resolution res)
   /**
    * Get instance of Generic that corresponds to this type parameter.
    *
-   * NYI: Since there is a 1-to-1 correspondance between type parameter features
+   * NYI: Since there is a 1-to-1 correspondent between type parameter features
```

Comment:
should be correspondence, imho?

Reply:
leo.org does not agree, we might need a native speaker to clarify this.